packages/turbo-repository/rust/src/lib.rs

```diff
@@ -63,4 +99,29 @@ impl Workspace {
     pub async fn find_packages(&self) -> std::result::Result<Vec<Package>, napi::Error> {
         self.packages_internal().await.map_err(|e| e.into())
     }
+
+    #[napi]
+    pub async fn package_graph(&self) -> Result<HashMap<String, Vec<String>>, Error> {
```

Comment:
Nit with naming: I think we should make it clear that this method is returning a reversed package graph where each node gets mapped to all nodes that depend on it instead of all nodes that it depends on. If I called this for `A -> B -> C` and got `{ "A": [], "B": ["A"], "C": ["B"]}` I would be surprised.

Reply:
renamed